Index: MEGAsync-5.2.0.0_Linux/contrib/cmake/CMakeLists.txt
===================================================================
--- MEGAsync-5.2.0.0_Linux.orig/contrib/cmake/CMakeLists.txt
+++ MEGAsync-5.2.0.0_Linux/contrib/cmake/CMakeLists.txt
@@ -65,7 +65,7 @@ set (USE_LIBRAW 1 CACHE STRING "")
 set (USE_SODIUM 1 CACHE STRING "")
 set (USE_FFMPEG 1 CACHE STRING "")
 set (USE_PDFIUM 1 CACHE STRING "")
-set (USE_FREEIMAGE 1 CACHE STRING "")
+set (USE_FREEIMAGE 0 CACHE STRING "")
 set (USE_QT 1 CACHE STRING "")
 set (ENABLE_LOG_PERFORMANCE 1 CACHE STRING "")
 set (NO_READLINE 0 CACHE STRING "")
